### DISCUSSION TITLE
关于chatglm3-6b-base中confi.json的seq_length与8k不符问题

### DISCUSSION THREAD

**Discussion by Ye-XX** [zai-org/ChatGLM3#1158] [Bad Case]
github项目介绍中，显示chatglm3-6b-base的Seq Length是8k，但是chatglm3-6b-base项目小的config.json配置汇中，显示它的Seq Length是32k，这是介绍上出现错误，还是config文件出错？
![微信图片_20240422113954](https://github.com/THUDM/ChatGLM3/assets/42534288/f2336ee2-e86d-4550-8327-e373fdb5f54a)
![2](https://github.com/THUDM/ChatGLM3/assets/42534288/14dbfae5-2696-4244-9c41-048c4cb3c0c2)

**Comment by zRzRzRzRzRzRzR** [comment]
嗯，但是实际效果确实只有8k

**Reply by Ye-XX** [reply]
那这有点奇特了，按理说应该是以这个配置参数为标准才对
